payments-ui-core/src/main/java/com/stripe/android/ui/core/Amount.kt

```diff
@@ -19,6 +21,10 @@ data class Amount(val value: Long, val currencyCode: String) : Parcelable {
     fun buildPayButtonLabel() =
         resolvableString(
             R.string.stripe_pay_button_amount,
-            CurrencyFormatter.format(value, currencyCode)
+            CurrencyFormatter.format(
+                amount = value,
+                amountCurrencyCode = currencyCode,
+                targetLocale = AppCompatDelegate.getApplicationLocales()[0] ?: Locale.getDefault()
```

Comment:
You might want first or null or something instead.

Reply:
`getApplicationLocales` returns `LocaleListCompat` which doesn't have the usual list extension methods. If the `LocalListCompat` is empty `get(0)` will return null and not throw and IndexOutOfBounds exception
```
@Override
public Locale get(int index) {
    return (0 <= index && index < mList.length) ? mList[index] : null;
}
```